flujo/agents/repair.py

```
@@ -118,8 +118,25 @@ def _is_json(text: str) -> bool:
         obj = data if not isinstance(data, str) else safe_deserialize(json.loads(data))
-        # Use enhanced serialization for better handling of complex objects
-        serialized = safe_serialize(obj)
+
+        def _serialize(obj: Any) -> Any:
+            try:
+                from pydantic import BaseModel as _BM
+
+                if isinstance(obj, _BM):
+                    return obj.model_dump(mode="json")
+            except Exception:
+                pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)